.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,25 @@
+name: NPM publish release
```

Comment:
This [example](https://github.com/actions/starter-workflows/blob/main/ci/npm-publish.yml) creates a build job which I like - what do you think? I'm okay moving ahead as is however.

Reply:
I'm not opposed, but don't think it's necessarily _better_ in the case where we only publish to npm